main.py

```diff
@@ -1,2 +1,4 @@
 if __name__ == '__main__':
     pass
+
```

Comment:
??? 'why is the main here? why is it in git? dont use git add -a ever, use git add -p